### DISCUSSION TITLE
[Feature] Show all shared links for a particular album only

### DISCUSSION THREAD

**Discussion by Snowknight26** [immich-app/immich#9989] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

For this feature request, any time the word "album" is used it can literally mean an album or a set of assets not in an album, since it applies to both scenarios.

When viewing a shared album, such as via Sharing, there's no intuitive way to know whether a shared link (or multiple links) for this particular album has been made before. The list of shared links available through `Sharing -> Shared links` lists _all_ shared links,.

Having a way to see any existing shared links _only_ for the specific album would make managing shared links/albums much easier and more user-friendly.

Currently there's no easy way to determine the following:
* has the album been shared already?
* what are all the shared links for this album?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
This is possible these days
